docs/_static/vcr/vcr_v2.yaml

```diff
@@ -416,7 +416,39 @@ paths:
                 $ref: "#/components/schemas/VerifiablePresentation"
         default:
           $ref: '../common/error_response.yaml'
+  /internal/vcr/v2/holder/{did}/vc:
+    post:
+      summary: Load a VerifiableCredential into the holders wallet.
+      description: |
+        If a VerifiableCredential is obtained in another way or when it's created without publishing, this API allows to add it to a wallet.
+        The DID of the holder has to be provided in the path.
+        It's assumed that the credentialSubject.id equals the holder DID.
 
+        error returns:
+        * 400 - Invalid credential
```

Comment:
or holder DID not managed by this node?

Reply:
The current wallet does not group credentials by wallet. If you add a VC that is not owned, then you'll never be able to use it.

Reply:
It does group them (on a shelf, identified by the credential subject DID), but without ownership check. So probably OK (we can always add the check later)